.travis.yml

```
@@ -15,9 +15,9 @@ notifications:
 
 env:
   matrix:
-    - ROS_DISTRO=melodic  ROS_REPO=ros
-    - ROS_DISTRO=melodic  ROS_REPO=ros-testing
-    - ROS_DISTRO=melodic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
+    - ROS_DISTRO=noetic  ROS_REPO=ros
+    - ROS_DISTRO=noetic  ROS_REPO=ros-testing
+    - ROS_DISTRO=noetic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_controllers/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
```

If https://github.com/ros-controls/ros_control/pull/478 works, after merging https://github.com/ros-controls/ros_control/pull/475, you can apply the following suggestion and remove the .rosinstall file form this repository:
```suggestion
    - ROS_DISTRO=noetic  UPSTREAM_WORKSPACE='https://raw.githubusercontent.com/ros-controls/ros_control/$ROS_DISTRO-devel/ros_control.rosinstall -ros_controllers'
```

---

applied